src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -532,6 +542,13 @@ public static ImmutableArray<TypedConstant> Encode(TypeSymbol type, TypeSymbol s
                 return names;
             }
 
+            private static bool TryGetNames(TypeSymbol type, ArrayBuilder<string> namesBuilder)
+            {
+                type.VisitType((t, builder, _ignore) => AddNames(t, builder), namesBuilder);
+                Debug.Assert(namesBuilder.Any());
+                return namesBuilder.All(name => name == null);
```

Comment:
`TryGetNames` should probably return false if all names are `null`. (Sorry, that was my mistake in earlier comment.)